rcongui/src/components/SettingsView/randomBroadcasts.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { toast } from "react-toastify"
+import "react-toastify/dist/ReactToastify.css";
+import { showResponse } from "../../utils/fetchUtils";
+import InputLabel from "@material-ui/core/InputLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import Grid from "@material-ui/core/Grid";
+
+let active = false;
```

Comment:
Hmm, why do you need a non-constant variable in the global scope? That does not seem right